README.md

```diff
@@ -128,95 +147,82 @@ or to copy over a file with clients defined, set the hiera variable:
 ``` yaml
 ---
 # The setting is
-# freeradius::v3::conf::clients_conf_source: <source for file>
-# For example if using a puppet source:
-freeradius::v3::conf::clients_conf_source: puppet:///modules/myconfigmod/freeradius/client.conf
+# freeradius::v3::conf::clients_conf_content: <exact content to add to file>
+freeradius::v3::conf::clients_conf_content: >
+  Your entire
+  configuration
+  goes here
 ```
 
 
 ### Other configuration
 
-The following configurations are not needed for the LDAP configuration.  These are a few
-examples of further configuration or alternate configuration that can be done.
-
-#### Add local radius users and trigger.
```

Comment:
Why was this example removed?

Reply:
@lnemsick-simp The example parameters were incorrect and I don't know the correct content to add to make this work properly. You have to know more about the `freeradius` configuration than a starting user would understand to make any advanced configuration work but it can be inferred from the rest of the documentation.